rktlet/cli/init.go

```diff
@@ -37,10 +37,14 @@ func NewSystemd(systemdRunPath string, execer utilexec.Interface) Init {
 
 // StartProcess runs the 'command + args' as a child of the init process,
 // and returns the id of the process.
-func (s *systemd) StartProcess(command string, args ...string) (id string, err error) {
+func (s *systemd) StartProcess(cgroupParent, command string, args ...string) (id string, err error) {
 	unitName := fmt.Sprintf("rktlet-%s", uuid.New())
 
 	cmdList := []string{s.systemdRunPath, "--unit=" + unitName}
+	if cgroupParent != "" {
+		// TODO(yifan): This requires the cgroup parent to nest under systemd cgroup.
```

Comment:
tiny nit, more correct terminology is probably "use the systemd subsystem vs older resource subgroups" or such